OpenEdXMobile/src/main/java/org/edx/mobile/view/RegisterActivity.java

```diff
@@ -285,13 +285,17 @@ public void onException(Exception ex) {
                             if (key.equalsIgnoreCase(fieldView.getField().getName())) {
                                 List<RegisterResponseFieldError> error = messageBody.get(key);
                                 showErrorOnField(error, fieldView);
-                                fieldErrorShown = true;
+                                if (!fieldErrorShown) {
```

Comment:
Actually, we're showing both, so we should probably change it to a generic name instead. This has been done in commit 77a4185fc315777ed4182346117f7c049488e366.